Plugins/org.mitk.gui.qt.application/src/internal/QmitkInfoDialog.cpp

```diff
@@ -37,7 +37,7 @@ QmitkInfoDialog::QmitkInfoDialog(const QList<mitk::DataNode::Pointer>& nodes, QW
   m_SearchButton = new QPushButton("Search (F3)", this);
   m_SearchButton->installEventFilter(this);
   m_TextBrowser = new QTextBrowser(this);
-  QPushButton* cancelButton = new QPushButton("Cancel", this);
+  QPushButton* m_CloseButton = new QPushButton("Close", this);
```

Comment:
I wouldn't use `m_`, since this is not a member variable.

Reply:
I think using an .ui file would be the right way to go. Also typical buttons like Cancel, and so on may be already available through the Dialog API of Qt instead of creating own buttons.

Reply:
> I wouldn't use `m_`, since this is not a member variable.

Oups, you are right, I did not see that. Yes, the prefix m_ is wrong

Reply:
> 
> 
> I think using an .ui file would be the right way to go. Also typical buttons like Cancel, and so on may be already available through the Dialog API of Qt instead of creating own buttons.

You are right, .ui files would be an option. However, I did not want to rewrite the widget but only fix a misleading label.